msteams-platform/sbs-gs-blazorupdate.yml

```diff
@@ -331,7 +331,8 @@ items:
     1. Enter the following in the **Provision** dialog:
 
        1. **Subscription name**: Visual Studio Enterprise Subscription. </br>
-       1. **Resource group**: Select **New** to add the resource group generated for your app.</br>
+       1. **Resource group**: Select **New** to add the resource group generated for your app.</br> For more information see [Create resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal#create-resource-groups)
```

Comment:
Check if we need a link to create resource group? If the user can create a resource group by following the steps in the UI, this is not required.

Reply:
Yes the user can follow the steps in the UI, hence the link is not required, removed as suggested.